### PR TITLE
Feature: About gig module

### PR DESCRIPTION
**Pull request by JakubFrackowiak**
Add about gig module.

### CODE REVIEW

File: components/AboutGig/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      <Typography
+        variant="body1"
+        sx={{
+          columnCount: { xs: 1, sm: 1, md: 2, lg: 2, xl: 2 },
+          lineHeight: "1.8rem",
+          columnGap: 4,
+          columnWidth: "50%",
+          height: "900px",
+          color: "text.secondary",
+          p: { xs: "1rem", sm: "1rem", md: "1rem", lg: "0rem", xl: "0rem" },
```
Comment (piotrostr) [inline components/AboutGig/index.tsx:54]: if there are 0rem, might skip them

File: components/AboutGig/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        </Typography>
+      </Stack>
+      <Typography
+        variant="body1"
+        sx={{
+          columnCount: { xs: 1, sm: 1, md: 2, lg: 2, xl: 2 },
+          lineHeight: "1.8rem",
+          columnGap: 4,
+          columnWidth: "50%",
+          height: "900px",
```
Comment (piotrostr) [inline components/AboutGig/index.tsx:52]: why fixed 900px height?

File: components/AboutGig/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          color: "text.secondary",
```
Comment (piotrostr) [inline components/AboutGig/index.tsx:53]: color should not be in sx, might be just a prop

**Review by piotrostr — CHANGES_REQUESTED** [review]
please check in with Oliwier, he will provide you with introductory review and once you pass Oli's review I'll have another look

File: components/AboutGig/AboutGig.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import { Typography } from "@mui/material"
+import { Stack } from "@mui/system"
```
Comment (OliwierOST) [inline components/AboutGig/AboutGig.tsx:2]: import every MUI component from `@mui/material`

**Review by piotrostr — CHANGES_REQUESTED** [review]

**Review by piotrostr — APPROVED** [review]

**Review by OliwierOST — APPROVED** [review]

**Pull request merged**